config.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+
+	hcl "github.com/hashicorp/hcl"
+)
+
+// TTYShareConfig - configuration options for CLI flags or config file
+type TTYShareConfig struct {
+	LogFileName    string `json:"logFileName"`
+	UseTLS         bool   `json:"useTLS"`
+	Server         string `json:"server"`
+	configFileName string
+	commandName    string
+	commandArgs    string
+	versionFlag    bool
+}
+
+// Load - Load configuration from config file if available and parse CLI options
+// CLI options override any config file settings, having config file is useful
+// for changing default options
+func (conf *TTYShareConfig) Load() {
+	defConfigFileName := os.Getenv("HOME") + "/." + path.Base(os.Args[0])
```

Comment:
Please consider following the XDG Base Directory Specification on where to save configuration files!
https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html